_posts/2020-03-11-sr2020-covid-19-updates.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: news
+title: COVID-19 Coronavirus Student Robotics Updates
+---
+
+As you will be aware, the COVID-19 situation is quickly evolving, having recently been delcared a pandemic. 
+Many large events are being cancelled and we want to be public about how we are responding to the latest news.
+
+The Competition Team are working closely with the Student Robotics trustees so that we can make the most sensible decision.
```

Comment:
The specific detail about the different teams within Student Robotics feels a little unnecessary.